connectors/src/connectors/confluence/temporal/workflows.ts

```diff
@@ -0,0 +1,121 @@
+import type { ModelId } from "@dust-tt/types";
+import {
+  executeChild,
+  proxyActivities,
+  workflowInfo,
+} from "@temporalio/workflow";
+
+import type * as activities from "@connectors/connectors/confluence/temporal/activities";
+import type { DataSourceConfig } from "@connectors/types/data_source_config";
+
+// The Temporal bundle does not support the use of aliases in import statements.
+import { makeConfluenceSpaceSyncWorkflowIdFromParentId } from "./utils";
+
+const {
+  confluenceGetSpaceNameActivity,
+  confluenceListPageIdsInSpaceActivity,
+  confluenceSaveStartSyncActivity,
+  confluenceUpsertPageActivity,
+  confluenceSaveSuccessSyncActivity,
+
+  fetchConfluenceConfigurationActivity,
+  getSpaceIdsToSyncActivity,
+} = proxyActivities<typeof activities>({
+  startToCloseTimeout: "20 minutes",
+});
+
+export async function confluenceFullSyncWorkflow({
```

Comment:
Can you not export that function or put it in a non activity file because it might be used by a workflow without realizing that the token is leaked to Temporal?

Reply:
Mistake on my side. Sorry.